src/main/java/com/ceritoxi/euler2/Euler2.java

```diff
@@ -0,0 +1,21 @@
+package com.ceritoxi.euler2;
+
+import com.ceritoxi.util.Math;
+
+public class Euler2 {
+	
+	private int result;
```

Comment:
Right now this variable is completely useless. You should either feed the number (4000000) from the outside in the constructor:
```
public Euler2(int number) {
this.result = number;
...
```

OR, much better, do not use the result variable at all: Just feed the run function with an argument from the outside:
```
public void run(int number) {
println(sumOfEvenFibonacciTermsUnder(number));
}
```
Note: you can static import both println and sumOfEvenFibonacciTermsUnder for easier readbility

Reply:
Run the function twice? That would need some convincing.

Reply:
The '400000' is not the result, but simply an argument closely related to the Euler2 task. This is why I thought I should include it in the class itself. 

Not having a 'result' variable would make me run the function twice, since I need the result for the test. I know the test itself is redundant since the 'Math' class is already tested, but you yourself said that I should test everything.

Reply:
okay fair enough